task/jq/0.1/jq.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
since `tekton.dev/v1` is already in place, can we move this task to new `apiVersion`?